src/main/java/com/mk/cti/controller/PhoneController.java

```diff
@@ -34,6 +34,24 @@
             phoneResponse.setLoanId(0L);
             phoneResponse.setAuthCode("A093756186");
         }
+        // phoneResponse.setAuthCode("A093756186");
+
+        return new ResponseEntity<>(phoneResponse, HttpStatus.OK);
+    }
+
+    @RequestMapping(value="{phoneNumber1}", method = RequestMethod.GET)
+    public ResponseEntity<PhoneResponse> getCustomerLoanInfo1(@PathVariable("phoneNumber1") String phoneNumber) {
+
+        PhoneResponse phoneResponse = phoneResponseService.findByPhoneNumber(phoneNumber);
+
+        if(phoneResponse == null)
+        {
+            phoneResponse = new PhoneResponse();
+            phoneResponse.setCustomerId(0L);
+            phoneResponse.setLoanId(0L);
+            phoneResponse.setAuthCode("A093756186");
+        }
+        // phoneResponse.setAuthCode("A093756186");
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=squid%3ACommentedOutCodeLine)